docs/content/guides/operator/indexer-stack-setup.mdx

```diff
@@ -294,6 +294,86 @@ $ sui-indexer-alt-consistent-store run \
 	<ImportContent source="examples/prod-config/consistent-store.toml" mode="code" />
 </details>
 
+## GraphQL server setup
+
+GraphQL server instances read data from the indexer db, the consistent store and the archival store. The number of instances is determined by the required read throughput.
+
+#### Hardware requirements
+
+- **CPU:** 8 cores
```

Comment:
```suggestion
- **CPU:** 4 cores per node
```

Reply:
I'll have to admit I trusted the numbers in @evan-wall-mysten 's original [doc](https://github.com/MystenLabs/sui/pull/23966/commits/b48abae30b7f79e09588dad192423f9b53d23d88) without checking them myself. Let me double check our prod config for this.